zuul.d/jobs.yaml

```diff
@@ -1114,3 +1114,10 @@
       Run integration tests of Kubernetes IN Docker against ARM64 architecture
     run: playbooks/kind-integration-test-arm64/run.yaml
     nodeset: ubuntu-xenial-arm64
+
+- job:
```

Comment:
Just a question: This job doesn't include the `nodeset` section, does it means this job perhaps run any nodes in env? or run in a default nodes?

I just worry about if it run in a CentOS unfortunately, the `apt install` in job would be invaild.

Reply:
now the default nodeset is `ubuntu-xenial`, because all the jobs inherits from the `base` job, which defining the nodeset as `ubuntu-xenial`

Reply:
Cool, found it, thanks

[1] https://github.com/theopenlab/openlab-zuul-jobs/blob/master/zuul.d/jobs.yaml#L22